subgraph.yaml

```diff
@@ -51,6 +51,41 @@ dataSources:
         - event: URI(string,indexed uint256)
           handler: handleURI
       file: ./src/mapping.ts
+  - kind: ethereum/contract
+    name: SupeRare
+    network: mainnet
+    source:
+      address: "0x41a322b28d0ff354040e2cbc676f0320d8c8850d"
+      abi: SupeRareV2
+      startBlock: 10147631
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - NftContract
+        - Nft
+      abis:
+        - name: SupeRare
+          file: ./abis/SupeRare.json
+        - name: SupeRareV2
+          file: ./abis/SupeRareV2.json
+      eventHandlers:
+        - event: AddToWhitelist(indexed address)
```

Comment:
Thanks for the PR!
As I see only `Transfer` event is used. Maybe it will be cleaner to remove all unused events?

Reply:
of course!

Reply:
Entity are different in each contract so a new handle.